### DISCUSSION TITLE
Windows build

### DISCUSSION THREAD

**Discussion by scorninpc** [scorninpc/php-gtk3#166] [General]
Lets maintain this discus about windows build

reference: https://github.com/scorninpc/php-gtk3/issues/164

**Comment by scribnermj** [comment]
I'm currently using this PHP-GTK compile on Windows. And it is working for me. I had to make two changes to the CSS to stop warnings when running generated apps. One was a misspelling of the word 'transparent', which was previously spelled as 'transparant'. There was also a CSS item that was missing the 'px' in a definition of a border. I remember that particular one was in gtk.css on line 1600.   Since I had no experience with GTK programming, I have struggled, but AI has been helpful at times.  

For now, I'm content to continue working on my application, but at some point, I would like to share it with friends and may need some advice on exactly how to do that.

**Reply by scorninpc** [reply]
warnings, especially style warnings, have been a problem since GTK 1, not a problem with PHP-GTK itself. Is this causing any problems? If so, this is a concern.

**Reply by scribnermj** [reply]
So far, I've only had the two very minor issues. I've made good progress.  I was able to make the minor edits directly to the CSS file in one case.  I'm not getting any warnings. So far as I know there are no performance issues. 

I know a little bit of PHP, but nothing about SDK, so it's been a steep learning curve.  The Ai have been a help, I request a template or advice on a thing and then I have an idea how to do it. Ai makes mistakes too though so it's still just a lot of learning.

**Reply by scorninpc** [reply]
I think you can see some examples here https://andor.com.br/php-gtk/cookbook

AI will not help you too much, coz all examples in PHP-GTK will give you GTK 1 or 2, and also GTK 3 has alot changes between one minor version to another
